src/fty_metric_compute.conf.in

```diff
@@ -1,3 +0,0 @@
-# create runtime directories for bios-agent-cm
```

Comment:
Ok, now I saw the individual commit with reasoning. I think it does make sense in a short-term solution to make things "just work" quickly, but is BS regarding longterm FOSS distribution management, where datadir etc. are configurable and templatable (no hardcoded paths in committed sources, ability to install proto area under DESTDIR, etc).